### PR TITLE
Add eslint for test files

### PR DESCRIPTION
**Pull request by knolleary**
## Description

Adds our standard eslint rules for the test files

### CODE REVIEW

File: test/unit/configmap_spec.js [hunk 1]
```diff
@@ -2,108 +2,105 @@ const should = require('should') // eslint-disable-line
...
         })
         describe('has broker', function () {
-            let mosquitto_conf
-            before( function () {
-                const brokerCM = configMaps.filter(cm => cm.metadata.name === 'flowforge-broker-config')[0]
-                mosquitto_conf = brokerCM.data['mosquitto.conf']
+            // let mosquittoConf
+            before(function () {
+                // const brokerCM = configMaps.filter(cm => cm.metadata.name === 'flowforge-broker-config')[0]
+                // mosquittoConf = brokerCM.data['mosquitto.conf']
```
Comment (hardillb) [inline test/unit/configmap_spec.js:64]: Why were these removed? I assume because the test that makes use of it on lines 83-85 don't exist yet?

**Review by hardillb — CHANGES_REQUESTED** [review]
Should probably add eslint (and any rule sets) to the devDependencies of the tests package.json as well

File: test/unit/configmap_spec.js (hunk 1 above)
Comment (knolleary) [reply test/unit/configmap_spec.js:64]: > Why were these removed?

Flagged as unused variables and I couldn't see anything that hinted at them being needed... but I did go for commenting out rather than removing so they can be easily restored.

**Review by hardillb — APPROVED** [review]
